docs/quick-lint-js.schema.config

```diff
@@ -0,0 +1,65 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+
+    "title": "quick-lint-js configuration",
+    "description": "The quick-lint-js CLI and quick-lint-js editor plugins, can be configured using a quick-lint-js.config file.",
+    "$comment": "Refer https://quick-lint-js.com/config for more details.",
+
+    "type": "object",
+
+    "properties": {
+        "globals": {
+            "description": "Global variables which programs can use. It tells quick-lint-js what global variables to assume exist.",
+            "$comment" : "If the global variables you want are from a popular platform or library, you might want to use 'global-groups' instead.",
```

Comment:
`"$comment"` doesn't seem appropriate here. `"$comment"` is for authors if this file, not creators of quick-lint-js.config files:

> Implementations MUST NOT present this [`"$comment"`] string to end users.
https://json-schema.org/draft/2020-12/json-schema-core.html#section-8.3

I think this comment belongs in `"description"`.